README.md

```diff
@@ -48,6 +51,26 @@ To run the application in `development` mode run:
 npm run dev
 ```
 
+### Aws localstack
+
+#### Install and run LocalStack AWS
+
+- Install [LocalStack AWS CLI](https://github.com/localstack/awscli-local)
```

Comment:
Could add install localstack CLI https://docs.localstack.cloud/getting-started/installation/#localstack-cli

Reply:
Also I would not recommend the awscli-local. As it needs AWS CLI v1 which is old.

Reply:
I just add an alias, in fish not bash
```
function awslocal
  aws --profile localstack $argv --endpoint-url http://localhost:4566
end
```
I can see if change that separately in the readme :)